### ISSUE TITLE
[@nativescript/picker] Angular 12 compatibility

### ISSUE THREAD

**Issue by alexander-mai** [NativeScript/plugins#143]
The abstract class `TemplatedItemsComponent` is no longer available in `@ nativescript/angular` in version 12.

The following import is causing the error:
https://github.com/NativeScript/plugins/blob/a3729499fd9cc897cbd8ca7363b5a0deeb4d898b/packages/picker/angular/picker.directive.ts#L2

Thjis results in the following error message
```
WARNING in ./node_modules/@nativescript/picker/__ivy_ngcc__/angular/fesm2015/nativescript-picker-angular.js 62:35-58
export 'TemplatedItemsComponent' (imported as 'TemplatedItemsComponent') was not found in '@nativescript/angular' (possible exports: APP_ROOT_VIEW, ActionBarComponent, ActionBarScope, ActionItemDirective, AppHostAsyncView, AppHostView, BasePortalOutlet, BaseValueAccessor, COMMON_PROVIDERS, CdkPortal, CdkPortalOutlet, CheckedValueAccessor, CommentNode, ComponentPortal, DEVICE, DISABLE_ROOT_VIEW_HANDLING, DateValueAccessor, DetachedLoader, DomPortal, ENABLE_REUSABE_VIEWS, EmulatedRenderer, FrameDirective, FramePageComponent, FramePageModule, FrameService, InjectableAnimationEngine, InvisibleNode, ItemContext, ListViewComponent, ModalDialogParams, ModalDialogService, NAMESPACE_FILTERS, NATIVESCRIPT_MODULE_PROVIDERS, NATIVESCRIPT_MODULE_STATIC_PROVIDERS, NATIVESCRIPT_ROOT_MODULE_ID, NATIVE_DIALOG_DATA, NATIVE_DIALOG_DEFAULT_OPTIONS, NSEmptyOutletComponent, NSFileSystem, NSLocationStrategy, NSRouteReuseStrategy, NSRouterLink, NSRouterLinkActive, NativeDialogCloseDirective, NativeDialogConfig, NativeDialogModule, NativeDialogRef, NativeDialogService, NativeModalRef, NativeScriptAnimationDriver, NativeScriptAnimationPlayer, NativeScriptAnimationsModule, NativeScriptCommonModule, NativeScriptDocument, NativeScriptDomPortalOutlet, NativeScriptFormsModule, NativeScriptHttpClientModule, NativeScriptLoadingService, NativeScriptModule, NativeScriptNgSafeEvent, NativeScriptRendererFactory, NativeScriptRouterModule, NativeScriptSanitizer, NativescriptXhrFactory, NavigationButtonDirective, NgViewRef, NsHttpBackEnd, NsTemplatedItem, NumberValueAccessor, Outlet, PAGE_FACTORY, PageDirective, PageRoute, PageRouterOutlet, PageService, PlatformNamespaceFilter, Portal, PortalModule, RootCompositeModule, RootViewProxy, RouterExtensions, START_PATH, SelectedIndexValueAccessor, TabViewDirective, TabViewItemDirective, TemplateKeyDirective, TemplatePortal, TextNode, TextValueAccessor, TimeValueAccessor, _NativeDialogBase, createKeyframeAnimation, customFrameComponentFactory, customFrameDirectiveFactory, customPageFactory, customPageFactoryFromFrame, dashCaseToCamelCase, defaultNavOptions, defaultPageFactory, defaultPageFactoryProvider, detachViewFromParent, disableRootViewHanding, errorHandler, extractSingleViewRecursive, frameMeta, generateDetachedLoader, generateFallbackRootView, generateNativeScriptView, generateRandomId, generateRootLayoutAndProxy, getFirstNativeLikeView, getItemViewRoot, getSingleViewRecursive, getViewClass, getViewMeta, instantiateDefaultStyleNormalizer, instantiateRendererFactory, instantiateSupportedAnimationDriver, isBlank, isContentView, isDetachedElement, isInvisibleNode, isKnownView, isLayout, isPresent, isView, onAfterLivesync, onBeforeLivesync, once, platformNativeScript, platformNativeScriptDynamic, postAngularBootstrap$, preAngularDisposal$, provideLocationStrategy, registerElement, registerNativeScriptViewComponents, runNativeScriptAngularApp, throwIfAlreadyLoaded, throwNoPortalAttachedError, throwNullPortalError, throwNullPortalOutletError, throwPortalAlreadyAttachedError, throwPortalOutletAlreadyDisposedError, throwUnknownPortalTypeError, ɵCOMPONENT_VARIABLE, ɵCONTENT_ATTR, ɵHOST_ATTR, ɵViewUtil, ɵa, ɵactionBarMeta, ɵb, ɵc, ɵd, ɵe, ɵelementMap, ɵf, ɵg, ɵh, ɵi, ɵisActionItem, ɵisNavigationButton, ɵj, ɵk, ɵl, ɵm, ɵn, ɵo)
```

**Comment by justin-daniel** [comment]
It looks like this was fixed but maybe never released? https://github.com/NativeScript/plugins/pull/147/files#diff-59b9fd7cf35f03479973d3245b9025602d3b1b7bc1c528624a697c7ff5ce3e84

**Comment by NathanWalker** [comment]
oh my goodness @justin-daniel you're right, thank you for the ping! Published 4.0.0 just now with angular 12 compat (major version bump since it's breaking change for anyone on angular 11 and below)
